src/main/resources/data/testdata1.csv

```diff
@@ -1,4 +1,4 @@
-,trainingid,firstname,lastname,ssn,sex,dob,race,ethnicity
+index,trainingid,firstname,lastname,ssn,sex,dob,race,ethnicity
```

Comment:
Why do you need index here?

Reply:
This column serves as the primary key on the entities, just making sure every row gets its own entity.

Reply:
Isn't that the training id?

Reply:
A little more explanation:
This is the actual generated data for socrates.  There are 10 records/variants per unique person.  `trainingid` is a unique identifier for the person, `index` is the unique identifier for the record.
We wanna integrate by `index` to indicate individual entities.
After perfect linking, all entities within the same linked entity should have the same trainingid.

Reply:
👍